src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -161,6 +161,13 @@ private void handleAction(String event,
             } else if (StringUtils.equalsIgnoreCase("pull_request", event)) {
 
                 pr = getPullRequest(payload, gh);
+                GHIssueState state = pr.getPullRequest().getState();
```

Comment:
I see `GHIssueState state = comment.getIssue().getState();` is also called in the IF condition above

May be call once by moving outside IF statement?

Reply:
> I see `GHIssueState state = comment.getIssue().getState();` is also called in the first IF condition above
> 
> May be call once by moving outside IF statement?

A comment goes against an issue, and a PR is a type of issue.

But a `pull_request` event is only ever a PR, so they are different.

It would've been nicer if there could be a single check, because then the _existing_ tests would cover it, but I don't think it's possible.

Reply:
ah yes. I am blind. ignore. sorry